assets/js/src/common/subscribers_cache_message.tsx

```diff
@@ -6,14 +6,27 @@ type Props = {
   cacheCalculation: string;
 };
 
-const handleRecalculate = () => {
-  window.location.reload();
-};
-
-const SubscribersCacheMessage = ({ cacheCalculation }: Props): JSX.Element => {
+export function SubscribersCacheMessage({ cacheCalculation }: Props): JSX.Element {
   const [loading, setLoading] = useState(false);
   const datetimeDiff = new Date().getTime() - new Date(cacheCalculation).getTime();
   const minutes = Math.floor((datetimeDiff / 1000) / 60);
+
+  const handleRecalculate = () => {
+    setLoading(true);
+    MailPoet.Ajax.post({
+      api_version: MailPoet.apiVersion,
+      endpoint: 'settings',
```

Comment:
I am not sure why 'settings'. That doesn't sound to me like the best place for subscribers count recalculation.

Reply:
I thought that it is similar to the recalculation of the engagement score. So, I placed it in the settings. Which place is better for you?

Reply:
Right, I didn't realize that. The engagement score is there because we call it from the settings page. If you think this is the best place, let's leave it here. I thought maybe the subscribers API would be better or segments maybe.